server/server.js

```diff
@@ -37,7 +39,7 @@ app.get('/reviews/:id', (req, res) => {
 });
 
 const server = app.listen(app.get('PORT'), () => {
-  console.log(`listening on port ${server.address().port}`);
+  console.log(`${server.address(), listening on port ${server.address().port}`);
```

Comment:
What sets the server.address()? Would this come from the host? Would you use this to be able to route different requests? Is it accessible similar to an environmental variable?

Reply:
server.address() is built into node api but it should be server.address().address so i'm making a new commit. i don't think it can be changed but only reads the current address in use.
https://nodejs.org/api/net.html#net_server_address